streamwebs_frontend/streamwebs/views.py

```diff
@@ -24,11 +24,203 @@
+        'somewhat_sensitive_total', 'aquatic_worm', 'blackfly', 'leech',
+        'midge', 'snail', 'mosquito_larva', 'tolerant_total', 'wq_rating'
+    )
+    return render_to_csv_response(
+        macros, field_header_map={
+            'date_time': 'date/time', 'site__site_name': 'site',
+            'time_spent': 'time spent sorting',
+            'num_people': '# of people sorting', 'water_type': 'water type',
+            'riffle_beetle': 'riffle beetle', 'water_penny': 'water penny',
+            'sensitive_total': 'sensitive total',
+            'clam_or_mussel': 'clam/mussel', 'crane_fly': 'crane fly',
+            'somewhat_sensitive_total': 'somewhat sensitive total',
+            'aquatic_worm': 'aquatic worm', 'mosquito_larva': 'mosquito larva',
+            'tolerant_total': 'tolerant total',
+            'wq_rating': 'water quality rating'
+        }
+    )
+
+
+# Come back to this to fix schools and foreign key calls
+def export_ript(request, site_slug):
+    site = Site.objects.get(site_slug=site_slug)
+    rip_transect = RiparianTransect.objects.filter(site_id=site.id)
+
+    sheets = []
+    for each in rip_transect:
+        sheets.append(each.id)
+
+    zones = TransectZone.objects.filter(transect_id=sheets[0])
+    zones = zones.values(
+        'transect__school', 'transect__date_time', 'transect__site__site_name',
+        'transect__weather', 'transect__slope', 'transect__notes', 'zone_num',
+        'conifers', 'hardwoods', 'shrubs', 'comments'
+    )
+    sheets.pop(0)
+
+    for sheet in sheets:
+        n_zones = TransectZone.objects.filter(transect_id=sheet)
+        n_zones = n_zones.order_by('transect__id', 'zone_num').values(
+            'transect__school', 'transect__date_time',
+            'transect__site__site_name', 'transect__weather',
+            'transect__slope', 'transect__notes', 'zone_num', 'conifers',
+            'hardwoods', 'shrubs', 'comments'
+        )
+        zones = zones | n_zones
+
+    filename = 'riparian_transect_export'
+
+    return render_to_csv_response(
+        zones, filename=filename, field_header_map={
+            'transect__school': 'school', 'transect__date_time': 'date/time',
+            'transect__site__site_name': 'site',
+            'transect__weather': 'weather', 'transect__slope': 'slope',
+            'transect__notes': 'notes', 'zone_num': 'zone number'
+        }
+    )
+
+
+def export_cc(request, site_slug):
+    site = Site.objects.get(site_slug=site_slug)
+    canopyc = Canopy_Cover.objects.filter(site_id=site.id)
+
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; \
+                                       filename="canopy_cover_export.csv"'
+    writer = csv.writer(response)
+    writer.writerow(['school', 'date/time', 'site', 'weather', 'north',
+                     'east', 'south', 'west', 'estimated canopy cover'])
+
+    sheets = []
+    for each in canopyc:
```

Comment:
I think you can just combine this into one loop and not use the `sheets` variable.